### ISSUE TITLE
Improve label quality rendering with signed distance fields

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium#5022]
This will be especially useful when the label is scaled to be bigger/smaller than the rasterized glyphs in the texture atlas.

Resources from #1075:
* http://aras-p.info/blog/2017/02/15/Font-Rendering-is-Getting-Interesting/
* https://github.com/mattdesl/gl-sprite-text
* https://www.essentialmath.com/?p=111
* http://wdobbie.com/post/gpu-text-rendering-with-vector-textures/
* http://metalbyexample.com/rendering-text-in-metal-with-signed-distance-fields/

**Comment by pjcozzi** [comment]
@jasonbeverage this issue has my notes on SDF if you are looking for some reading. :smile:

**Comment by jasonbeverage** [comment]
Definitely, thanks!  It sounds pretty likely that I'll get a chance to work on this next year.  I'll keep you updated

**Comment by mramato** [comment]
From Don McCurdy via Twitter: https://msdf-bmfont.donmccurdy.com/

https://twitter.com/donrmccurdy/status/962497620663926784

**Comment by pjcozzi** [comment]
@jasonbeverage any chance you are still planning to do this / already started???

**Comment by jasonbeverage** [comment]
Hey Patrick,

I actually did start looking at this and did some promising experiments.
 I'm still working on getting funding lined up for it, along with some
other cesium related tasks.  Ill ping my customer again and see if we can
get something lined up.

Jason

**Comment by pjcozzi** [comment]
Very nice @jasonbeverage, please keep me posted so we can best support you!

**Comment by jasonbeverage** [comment]
Thanks a bunch, I'll keep you posted.  I'm hoping we can get started this
summer but you never know.

**Comment by ggetz** [comment]
Went to that course at SIGGRAPH and I would give it a :+1:

**Comment by OmarShehata** [comment]
This looks like a nice open source SDF library: https://github.com/mapbox/tiny-sdf

**Comment by jasonbeverage** [comment]
@OmarShehata indeed it is a nice little library, and I'll be including it in a PR hopefully today ;)
